src/app/client/src/app/modules/dashboard/components/re-issue-certificate/re-issue-certificate.component.ts

```diff
@@ -173,6 +173,11 @@ export class ReIssueCertificateComponent implements OnInit, OnDestroy {
     this.userName = this.userName ? this.userName.trim() : this.userName;
     const value = !_.isEmpty(this.userData) ? this.userData['courses'].batches = [] : [];
   }
+
+  enableReIssueCert(batch) {
```

Comment:
What will happen when both PDF & SVG certificates are exist? Which certificate will get re-issues that time?
 
We should issue both SVG & PDF certificates. Check this once.

Reply:
Certificate is being issued in this format:
SVG - SVG format
PDF - PDF format
SVG and PDF - SVG format